code/modules/client/preferences.dm

```diff
@@ -2,20 +2,26 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 /datum/preferences
 	var/client/parent
-	//doohickeys for savefiles
+	/// The path to the general savefile for this datum
 	var/path
-	var/default_slot = 1 //Holder so it doesn't default to slot 1, rather the last one used
+	/// Whether or not we allow saving/loading. Used for guests, if they're enabled
+	var/load_and_save
```

Comment:
can you set a default value of true or false

Reply:
ah fuck ty

Reply:
```suggestion
	var/load_and_save = TRUE
```